File: One-Umbrella.Domain/Entities/Grid.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OneUmbrella.Domain.Entities
{
    public class Grid
    {
        public int GridId { get; set; }
        public string? GridName { get; set; }
        public int RestaurantId { get; set; }
        public int GridRows { get; set; }
        public int GridColumns { get; set; }
    }
}

File: one-umbrella.client/src/app/shared/models/restaurant/grid.ts
import { StructuralElement } from "./structuralElement";
import { Table } from "./table";

export interface Grid {
    gridId : number,
    restaurantId : number,
    gridName : string,
    gridRows : number,
    gridColumns : number,
    gridTables : Table[],
    gridElements : StructuralElement[]
}